src/components/Film.vue

```diff
@@ -601,30 +595,33 @@ a:focus {
 
 .dark #advanced label {
 	background: #526e89;
+	transition: background ease-in-out 0.25s;
 }
 
 #advanced input[type="radio"]:checked + label {
 	background: #40bcf4;
+	transition: border-color ease-in-out 0.25s;
 	border-color: #000;
 	color: #fff;
 }
 
 .dark #advanced input[type="radio"]:checked + label {
 	border-color: #fff;
+	transition: border-color ease-in-out 0.25s;
 }
 
 #advanced input[type="radio"]:checked + label svg {
 	fill: #fff;
 }
 
 #advanced input[type="radio"]:focus + label {
```

Comment:
I would remove these two transitions (lines 619 and 624). While it does make the transition from light to dark slightly cleaner, it's a really small improvement—and the downside is that the box-shadow is animated when focusing on an option now (a pet peeve of mine).

Reply:
now that I'm thinking about it, it doesn't even improve the dark mode transition, because by clicking the switch the radio button will lose focus.

Reply:
done